src/logic/map_objects/tribes/worker.cc

```diff
@@ -1112,7 +1112,8 @@ Worker::Worker(const WorkerDescr& worker_descr)
      ware_economy_(nullptr),
      supply_(nullptr),
      transfer_(nullptr),
-     current_exp_(0) {
+     current_exp_(0),
+     is_destroying_(false) {
```

Comment:
Rename to `is_being_destroyed_`. This sounds as if the worker was destroying something else.